addon/components/horizontal-list-swiper/gesture/component.js

```diff
@@ -6,22 +6,34 @@ import Ember from 'ember';
 import RecognizerMixin from 'ember-gestures/mixins/recognizers';
 import layout from './template';
 import { run } from '@ember/runloop';
+import {getOwner} from '@ember/application';
```

Comment:
What happened to your `eslint` update to force `{ getOwner }`?

Reply:
@txm not using the addon for now. Will do - adding to bullet list now